src/test/java/step_definitions/WikipediaSteps.java

```diff
@@ -1,5 +1,6 @@
 package step_definitions;
 
+import core.BaseClass;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'core.BaseClass'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://engci-sonar-sjc.cisco.com/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)